runner/job_manager.py

```diff
@@ -413,3 +419,12 @@ def delete_resources(self, batch_service_client: batch.BatchExtensionsClient,
             logger.info(
                 "Job: {}. did not complete successfully, Container {} was not deleted.".format(
                     self.job_id, self.storage_info.output_container))
+
+    def update_params_with_values_from_keyvault(self, parameters, keyvault_client_with_url):
```

Comment:
>update_params_with_values_from_keyvault [](start = 8, length = 39)

This needs all the docs rtype/return/params ':Type' and -> type 

I would move this to the Utils.py file

Reply:
Done